### DISCUSSION TITLE
How does one disable console logs?

### DISCUSSION THREAD

**Discussion by hsoj48** [spring-projects/spring-shell#569] [Q&A] [answered]
When using logback for application logging within a spring-shell application, logs and stacktraces seem to be printed to the console by default.  This seems to cloud an interactive shell with information the user may not want to see.

How do we disable console logging when running the application?  What are the recommended conventions for printing lines to the shell and printing to a log file?

**Comment by jvalkeal** [comment, marked as answer]
It's mentioned in docs and sample also disable console logging and has lines commented out to write to a file. https://github.com/spring-projects/spring-shell/blob/main/spring-shell-samples/src/main/resources/application.yml.
